portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -21,22 +21,6 @@
 
 package org.sakaiproject.portal.charon.site;
 
-import java.net.URI;
```

Comment:
Organising the imports means this will never apply cleanly against Sakai and other cherry-picked patches won't apply cleanly. It's almost never worth doing this.